app/db/setting.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const setting = new Schema({
+  onEventMessage: {
+    type: String,
+    default: null
+  },
+  periodicMessage: {
+    type: String,
+    default: null
+  },
+  intervalInMilliseconds: {
+    type: Number,
+    default: 0
```

Comment:
Wasn't 0 sending continuously? How about a more reasonable default, like 10 seconds?

Reply:
Not sure where we ended up regarding this case - couldn't even find the related issue/PR yet.

However, you're right, setting the default interval to 10 seconds should be more than enough. :)